bin/setup

```diff
@@ -0,0 +1,36 @@
+#!/usr/bin/env ruby
```

Comment:
Just curious. 

Did you write this or was auto-generated?
I found it super useful

Reply:
No , it was auto-gererated   😄